beanie/odm/settings/base.py

```diff
@@ -20,7 +20,7 @@ class ItemSettings(BaseModel):
     union_doc: Optional[Type] = None
     union_doc_alias: Optional[str] = None
     class_id: str = "_class_id"
-
+    read_only: bool = False
```

Comment:
You have to choose or ItemSettings (it means it will be applied not only to Documents, but Views and etc) or DocumentSettings. Having this property in both is redundant